README.md

```diff
@@ -968,7 +968,7 @@ func main() {
 
 #### Iterating Over a Hash/Map
 
-##### Perl
+##### Perl using keys
```

Comment:
We could also add another sub-heading here or maybe put the description in parens: `Perl (using keys)`?

Reply:
IMHO the headings are not optimal, when they just say `Go` and `Perl`, but this is up to you.

Reply:
In the generated table of contents they do get a hierarchy, which is easy enough to scan, but I don't feel too strongly about it right now.